sdk/src/main/java/io/opentelemetry/sdk/trace/RecordEventsReadableSpanImpl.java

```diff
@@ -69,6 +69,8 @@
   private final Resource resource;
   // The start time of the span.
   private final long startNanoTime;
+  // Whether span is recording events
+  private final boolean recordEvents;
```

Comment:
This is not needed. `RecordEventsReadableSpanImpl. isRecordingEvents()` will always return true.